controller/src/beerocks/master/db/db.h

```diff
@@ -589,6 +589,158 @@ class db {
+     * 
+     * @return client time-life delay, value of 0 means not-configured.
+     */
+    std::chrono::seconds get_client_time_life_delay(const sMacAddr &mac);
+
+    /**
+     * @brief Set the client's stay-on-initial-radio.
+     * 
+     * @param mac MAC address of client.
+     * @param stay_on_initial_radio enable client stay on the radio it initially connected to.
+     * @param set_persistent_db if set to true, update the persistent-db (write-through), default is true.
+     * 
+     * @return true on success.
+     * @return false on failure.
+     */
+    bool set_client_stay_on_initial_radio(const sMacAddr &mac, bool stay_on_initial_radio,
+                                          bool set_persistent_db = true);
+
+    /**
+     * @brief Get the client's stay-on-initial-radio.
+     * 
+     * @param mac MAC address of client.
+     * 
+     * @return enable client stay on the radio it initially connected to.
+     */
+    beerocks::ePersistentParamBool get_client_stay_on_initial_radio(const sMacAddr &mac);
+
+    /**
+     * @brief Set the client's initial-radio.
+     * 
+     * @param mac MAC address of client.
+     * @param initial_radio_mac the mac of radio that the client has initially connected to.
+     * @param set_persistent_db if set to true, update the persistent-db (write-through), default is true.
+     * 
+     * @return true on success.
+     * @return false on failure.
+     */
+    bool set_client_initial_radio(const sMacAddr &mac, const sMacAddr &initial_radio_mac,
+                                  bool set_persistent_db = true);
+
+    /**
+     * @brief Get the client's initial-radio.
+     * 
+     * @param mac MAC address of client.
+     * 
+     * @return mac of radio that the client has initially connected to.
+     */
+    sMacAddr get_client_initial_radio(const sMacAddr &mac);
+
+    /**
+     * @brief Set the client's stay-on-selected-band.
+     * 
+     * @param mac MAC address of client.
+     * @param stay_on_selected_band: enable client stay on the selected band/bands.
+     * @param set_persistent_db: set-persistent-db flag - default is true (write-through).
+     * 
+     * @return true on success.
+     * @return false on failure.
+     */
+    bool set_client_stay_on_selected_band(const sMacAddr &mac, bool stay_on_selected_band,
+                                          bool set_persistent_db = true);
+
+    /**
+     * @brief Get the client's stay-on-selected-band.
+     * 
+     * @param mac MAC address of client.
+     * 
+     * @return enable client stay on the selected band/bands.
+     */
+    beerocks::ePersistentParamBool get_client_stay_on_selected_band(const sMacAddr &mac);
+
+    /**
+     * @brief Set the client's selected-bands.
+     * 
+     * @param mac MAC address of client.
+     * @param selected_bands: client selected band/bands. FREQ_UNKNOWN is considered as "not-configured"
+     * @param set_persistent_db: set-persistent-db flag - default is true (write-through).
```

Comment:
You should not add colons between the argument name to the description.
Second, does set_persistent_db means "Save configuration on persistent database"?
Because it is much more understandable (at least for me) to phrase it like that.

Reply:
done